src/reporters/json.ts

```diff
@@ -302,6 +395,32 @@ export default class JSONReporter extends BaseReporter {
         status,
         error,
       }) => {
+        const screenshots = await gatherScreenshots(
+          join(CACHE_PATH, 'screenshots')
+        );
+        if (screenshots.length > 0) {
+          screenshots.forEach(({ blocks, reference, step, blob_mime }) => {
+            for (let i = 0; i < blocks.length; i++) {
+              const block = blocks[i];
+              this.writeJSON({
+                type: 'screenshot/block',
+                _id: block.id,
+                blob: block.blob,
+                blob_mime,
+              });
+            }
+
+            this.writeJSON({
+              type: 'step/screenshot_reference',
+              journey,
+              step,
+              payload: {
+                screenshot_ref: reference,
```

Comment:
Let's pull this out of payload and put it at the top level. We want to index some of the values here, they'll be useful to defer debugging storage issues since we can run queries to determine the reuse of screenshot blocks. 

You'll just need to add a new root field `"synthetics.screenshot_ref"`  with the ref

Reply:
Sounds good, will do. We don't want it to be in `root_fields`?